controllers/gitopsservice_controller.go

```
@@ -224,7 +225,21 @@ func (r *ReconcileGitopsService) Reconcile(ctx context.Context, request reconcil
 		return result, err
 	}
 
-	return r.reconcilePlugin(instance, request)
+	OCPVersion, err := util.GetClusterVersion(r.Client)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	v1, err := version.NewVersion(OCPVersion)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	v2, err := version.NewVersion("4.15.0")
+	if v1.LessThan(v2) {
+		return reconcile.Result{}, err
```

@ciiay I don't think we need to return an error here, we can just have 

```
if !v1.LessThan(v2) {
     r.reconcilePlugin(instnace, request)
    }
```